### ISSUE TITLE
Wrong Value for UV

### ISSUE THREAD

**Issue by TokyoSU** [veeti512/wadblender#15]
If you select anything except "Full Texture Pages", in unity the model will have black line caused by the UV.

**Comment by veeti512** [comment]
Did you set filter mode to none on the properties tab of the texture image in unity? I can't work on the tool during this time so I can't check

**Comment by TokyoSU** [comment]
yes, more precisely it's "filter mode: point (no filter)" but that dont resolve the black line, if i use "Full Texture Pages" then the black line is solved.

**Comment by TokyoSU** [comment]
dont tested the mode 256x256 but i think it work maybe

**Comment by veeti512** [comment]
I tried and that's uv bleeding.  I think that you can get rid of it completely only by doing a proper uv unwrap and rearranging the textures.
